packages/x-data-grid/src/utils/keyboardUtils.ts

```diff
@@ -57,11 +57,23 @@ export const isHideMenuKey = (key: React.KeyboardEvent['key']) => isTabKey(key)
 export function isPasteShortcut(event: React.KeyboardEvent) {
   if (
     (event.ctrlKey || event.metaKey) &&
-    event.key.toLowerCase() === 'v' &&
```

Comment:
not sure what the implications are ... we would definitely need some tests to secure this change

Reply:
it is the same issue as the CTRL+C shortcut, i will fix the test cases and push the changes

Reply:
Related to https://github.com/mui/mui-x/pull/3660/files#r793084149

Reply:
Ok, so it seems that I broke it with https://github.com/mui/mui-x/pull/9220/files#r1216831312. 

I can reproduce the bug 

<img width="387" alt="SCR-20240817-ufoh" src="https://github.com/user-attachments/assets/44dc39dd-2c74-41be-a654-5229267e3861">

and only `String.fromCharCode(event.keyCode)` seems to be usable to detect the shortcut correctly.